xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp

```diff
@@ -607,8 +607,31 @@ void CLinuxRendererGLES::RenderUpdateVideo(bool clear, DWORD flags, DWORD alpha)
     CDVDVideoCodecIMXBuffer *buffer = m_buffers[m_iYV12RenderBuffer].IMXBuffer;
     if (buffer != NULL && buffer->IsValid())
     {
-      ManageDisplay();
-      g_IMXContext.SetBlitRects(m_sourceRect, m_destRect);
+      RENDER_STEREO_MODE stereo_mode = g_graphicsContext.GetStereoMode();
+      if (stereo_mode)
+        g_graphicsContext.SetStereoView(RENDER_STEREO_VIEW_LEFT);
+
+     ManageDisplay();
+
+     if (stereo_mode = g_graphicsContext.GetStereoMode())
+       g_graphicsContext.SetStereoView(RENDER_STEREO_VIEW_OFF);
```

Comment:
If I leave out ManageDisplay() completely. I get it going the right way, but subs are not correctly centered as ManageDisplay calls CalcNormalSomething that cares for this.